script2.js

```diff
@@ -0,0 +1,121 @@
+    {
+      myOutputValue = "Please input a valid choice.";
+      return myOutputValue;
+    }
+
+    if (input == "reverse")
+    {
+      gameMode = "reverse";
+      return "Game mode has been reversed!";
+    }
+    else if (input == "normal")
+    {
+      gameMode = "normal";
+      return "Game mode is normal.";
+    }
+
+    if (gameMode == "normal")
+    {
+      if (input == comChoice)
+      {
+        myOutputValue = "You chose: " + input;
+        myOutputValue += "<br>Computer chose: " + comChoice;
+        myOutputValue += "<br>DRAW!<br>";
+        draw++;
+        myOutputValue += "<br>Congratulations " + username + ", Wins: " + win + " " + "Losses: " + loss + " " + "Draws: " + draw;
+      }
+      else if ((input == "Scissors" && comChoice == "Paper") || (input == "Paper" && comChoice == "Stone") || input == "Stone" && comChoice == "Scissors")
+      {
+        myOutputValue = "You chose: " + input;
+        myOutputValue += "<br>Computer chose: " + comChoice;
+        myOutputValue += "<br>WIN!!<br>";
+        win++;
+        myOutputValue += "<br>Congratulations " + username + ", Wins: " + win + " " + "Losses: " + loss + " " + "Draws: " + draw;
+      }
+      else
+      {
+        myOutputValue = "You chose: " + input;
+        myOutputValue += "<br>Computer chose: " + comChoice;
+        myOutputValue += "<br>LOSE!<br>";
+        loss++;
+        myOutputValue += "<br>Congratulations " + username + ", Wins: " + win + " " + "Losses: " + loss + " " + "Draws: " + draw;
+      }
+    }
+    else if (gameMode == "reverse")
+    {
+      if (input == comChoice)
+      {
+        myOutputValue = "You chose: " + input;
+        myOutputValue += "<br>Computer chose: " + comChoice;
+        myOutputValue += "<br>DRAW!<br>";
+        draw++;
+        myOutputValue += "<br>Congratulations " + username + ", Wins: " + win + " " + "Losses: " + loss + " " + "Draws: " + draw;
+      }
+      else if ((input == "Scissors" && comChoice == "Paper") || (input == "Paper" && comChoice == "Stone") || input == "Stone" && comChoice == "Scissors")
+      {
+        myOutputValue = "You chose: " + input;
+        myOutputValue += "<br>Computer chose: " + comChoice;
+        myOutputValue += "<br>LOSE!!<br>";
+        loss++;
+        myOutputValue += "<br>Congratulations " + username + ", Wins: " + win + " " + "Losses: " + loss + " " + "Draws: " + draw;
+      }
+      else
+      {
+        myOutputValue = "You chose: " + input;
+        myOutputValue += "<br>Computer chose: " + comChoice;
+        myOutputValue += "<br>WIN!<br>";
+        win++;
+        myOutputValue += "<br>Congratulations " + username + ", Wins: " + win + " " + "Losses: " + loss + " " + "Draws: " + draw;
+      }
+    }
+
+    
+    return myOutputValue;
+  }
+
+  
+};
+
+var computerChoice = function()
+{
+  var comChoice = Math.floor(1 + Math.random() * 3);
+  console.log("Com Choice: " + comChoice);
```

Comment:
Good job in converting random number to string!

Reply:
thanks for the comments and help thus far